src/Schema/ContextModule.ts

```diff
@@ -31,20 +36,28 @@ export enum ContextModule {
   liveAuctionsRail = "liveAuctionsRail",
   mainCarousel = "mainCarousel",
   minimalCTABanner = "minimalCTABanner",
+  newWorksByArtistsYouFollowRail = "newWorksByArtistsYouFollowRail",
   otherWorksByArtistRail = "otherWorksByArtistRail",
   otherWorksFromPartnerRail = "otherWorksFromPartnerRail",
   otherWorksFromShowRail = "otherWorksFromShowRail",
   otherWorksInAuctionRail = "otherWorksInAuctionRail",
   partnerHeader = "partnerHeader",
   pastFairs = "pastFairs",
+  popularArtistsRail = "popularArtistsRail",
```

Comment:
We also have `worksByPopularArtistsRail` on #61, is it possible these are duplicates?

Reply:
`popularArtistsRail` is one i added in reference to this rail on Eigen's home screen: 
![popularartists](https://user-images.githubusercontent.com/26879583/80118052-e5a91800-8555-11ea-8fad-8deb2bd6c980.png)

and i would expect this to be different than a rail of artworks that are by popular artists, hence the separation. I can for sure add descriptive comments throughout!

Reply:
I'm actually now wondering if `popularArtistsCardRail` is the better term, since this is a rail of cards (per the design).

Reply:
@samchieng is there a situation in which an auction/fair/artist card will standalone and not sit within a rail?

Reply:
This rail on iOS is actually the same as the "Most Popular" tab in `artistsToFollowRail` context_module on the home page on web. I also see a Trending artists rail in iOS that is the same as the "Trending" tab in the `artistsToFollowRail` on web.

<img width="667" alt="Screen Shot 2020-04-23 at 2 28 20 PM" src="https://user-images.githubusercontent.com/12847213/80135839-d387a380-856e-11ea-9a64-73cc8aa8615d.png">

This brings up an interesting point around parity between web and iOS. What are our thoughts on removing the `artistsToFollowRail` context_module in favor of the `trendingArtistsRail` and `popularArtistsRail` context_modules that Mike added? @eessex this would require us to update the context_module for a couple of `authImpression` events.

And then @mikehrom it might be helpful for us to compare the iOS rails to the logged-in (and logged-out) home page experiences to make sure our wording makes sense so that we can then use the context_modules you're adding now for web in the future. I'd be happy to pair with you today or tomorrow if you'd like!

Reply:
I'm open to sharing these contexts - feel free to add to the schema for `AuthContextModule` and I can remove the old ones in a separate PR.

Reply:
> @samchieng is there a situation in which an auction/fair/artist card will standalone and not sit within a rail?

It's something we're potentially thinking of having in the future, but nothing spec'd out yet.